### ISSUE TITLE
False error "The function parameters must match those in its docblock @param" when using local variable tag

### ISSUE THREAD

**Issue by profidocs** [PHPCheckstyle/phpcheckstyle#89]
Hi,

I've been successfully using phpcheckstyle. Thank you for the great work.
I noticed today the following: if you tag a local variable with a type, then the whole function reports incorrect parameters. Let me give you an example:
The code below will show: _The function test parameters must match those in its docblock @param_. 

```php
<?php
class Example
{
    /**
    * @param string $param1
    * @param string $param2
    * @return void 
    **/
    public function test($param1, $param2)
    {
        /** @var SomeClass | null */
        $someObject = null;
    }
}
```

To resolve the error, I must remove the tag from the local variable. This will pass:

```php
<?php
class Example
{
    /**
    * @param string $param1
    * @param string $param2
    * @return void 
    **/
    public function test($param1, $param2)
    {
        $someObject = null;
    }
}
```
It looks as if somehow the @var tag is added to the list of parameters already defined by the function ($param1, $param2). As a consequence, the linter reports a mismatch.
I need the initial tag, so how could I work around this issue? Any thoughts?

**Comment by tchule** [comment]
Hello,

It looks like a bug, I will have a look.

**Comment by profidocs** [comment]
Ok, thanks. I can confirm it works on my end. Let me know if I can help. The above is a simplified example, I can provide some more test cases if needed.
Regards

**Comment by tchule** [comment]
OK cool, I close the bug, thx.
